OpenRA.Mods.Common/Widgets/Logic/MissionBrowserLogic.cs

```diff
@@ -314,12 +314,13 @@ void SelectMap(MapPreview preview)
 				var speeds = modData.Manifest.Get<GameSpeeds>().Speeds;
 				gameSpeed = "default";
 
-				gameSpeedButton.GetText = () => speeds[gameSpeed].Name;
+				var speedText = new CachedTransform<string, string>(speed => modData.Translation.GetString(speed));
+				gameSpeedButton.GetText = () => speedText.Update(speeds[gameSpeed].Name);
```

Comment:
This can be
```suggestion
				gameSpeedButton.GetText = () => speedText.Update(gameSpeed);
```
with the `speeds[gameSpeed]` lookup inside the `CachedTransform`.